### PR TITLE
SC-2305

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

**Review by amit-tarento — COMMENTED** [review]
@Hari-stackroute Please check the SearchController and controller application.conf file
you reverted back the changes, merge it correctly with release-4.3.0

File: service/src/main/java/org/sunbird/util/Util.java [hunk 1]
```diff
@@ -188,6 +191,16 @@ public static SearchDTO createSearchDto(Map<String, Object> searchQueryMap) {
     return search;
   }
 
+  /**
+   * if Object is not null then it will return true else false.
+   *
+   * @param obj Object
+   * @return boolean
+   */
```
Comment (amit-tarento) [inline service/src/main/java/org/sunbird/util/Util.java:199]: remove this method

File: service/src/main/resources/application.conf [hunk 2]
```diff
@@ -0,0 +1,590 @@
+SunbirdMWSystem {
+  default-dispatcher {
+    type = "Dispatcher"
+    executor = "fork-join-executor"
+    fork-join-executor {
+      parallelism-min = 8
+      parallelism-factor = 32.0
+      parallelism-max = 64
```
Comment (amit-tarento) [inline service/src/main/resources/application.conf:8]: delete this file , not needed any more as we moved this under controller/application.conf

File: service/src/main/java/org/sunbird/actor/user/UserBaseActor.java [hunk 3]
```diff
@@ -38,6 +38,8 @@
 
   protected UserLookupService userLookupService = UserLookUpServiceImpl.getInstance();
   protected LocationClient locationClient = LocationClientImpl.getInstance();
+  protected UserCreateRequestValidator userCreateRequestValidator =
```
Comment (amit-tarento) [inline service/src/main/java/org/sunbird/actor/user/UserBaseActor.java:41]: if all method inside is static then no need to create object for this class

Comment (Hari-stackroute) [reply service/src/main/java/org/sunbird/actor/user/UserBaseActor.java]: done

File: service/src/main/resources/application.conf (hunk 2 above)
Comment (Hari-stackroute) [reply service/src/main/resources/application.conf]: done

File: service/src/main/java/org/sunbird/util/Util.java (hunk 1 above)
Comment (Hari-stackroute) [reply service/src/main/java/org/sunbird/util/Util.java]: done

File: service/src/main/java/org/sunbird/service/user/impl/UserOrgServiceImpl.java [hunk 4]
```diff
@@ -22,6 +31,26 @@ public static UserOrgService getInstance() {
...
+    reqMap.put(JsonKey.ORG_JOIN_DATE, ProjectUtil.getFormattedDate());
+    reqMap.put(JsonKey.IS_DELETED, false);
+    reqMap.put(JsonKey.ASSOCIATION_TYPE, userMap.get(JsonKey.ASSOCIATION_TYPE));
+    if (StringUtils.isNotEmpty((String) userMap.get(JsonKey.HASHTAGID))) {
+      reqMap.put(JsonKey.HASHTAGID, userMap.get(JsonKey.HASHTAGID));
+    }
+    Util.DbInfo usrOrgDb = Util.dbInfoMap.get(JsonKey.USER_ORG_DB);
+    try {
+      cassandraOperation.insertRecord(
+          usrOrgDb.getKeySpace(), usrOrgDb.getTableName(), reqMap, context);
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply service/src/main/java/org/sunbird/service/user/impl/UserOrgServiceImpl.java]: done

File: service/src/main/java/org/sunbird/service/organisation/impl/OrgExternalServiceImpl.java [hunk 5]
```diff
@@ -1,43 +1,97 @@
...
+  private final String KEYSPACE_NAME = JsonKey.SUNBIRD;
+  private final String ORG_EXTERNAL_IDENTITY = JsonKey.ORG_EXT_ID_DB;
   private OrgExternalDao orgExtDao = new OrgExternalDaoImpl();
-  private OrgDao orgDao = OrgDaoImpl.getInstance();
 
   @Override
   public String getOrgIdFromOrgExternalIdAndProvider(
       String externalId, String provider, RequestContext context) {
-    return orgExtDao.getOrgIdFromOrgExternalIdAndProvider(externalId, provider, context);
+    Map<String, Object> dbRequestMap = new LinkedHashMap<>(3);
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply service/src/main/java/org/sunbird/service/organisation/impl/OrgExternalServiceImpl.java]: done

**Review by reshmi-nair — APPROVED** [review]

**Pull request merged**